shopfloor/services/schema.py

```diff
@@ -57,6 +57,7 @@ def picking(self):
             "origin": {"type": "string", "nullable": True, "required": False},
             "note": {"type": "string", "nullable": True, "required": False},
             "move_line_count": {"type": "integer", "nullable": True, "required": True},
+            "move_lines": {"type": "list", "nullable": True, "required": False},
```

Comment:
I don't think we want this: is going to compute lines every time.
Add a flag instead (as per with_packaging on move_line)
And do the same for `data.picking`

Reply:
Just checked: it's better if you use `data_detail.picking` which already includes this (and its companion `schema_detail.picking`) :wink: